net/unbound/patches/001-conf.patch

```diff
@@ -1,133 +1,94 @@
- 	# the number of slabs must be a power of 2.
- 	# more slabs reduce lock contention, but fragment memory usage.
- 	# infra-cache-slabs: 4
-+	infra-cache-slabs: 1
- 
- 	# the maximum number of hosts that are cached (roundtrip, EDNS, lame).
- 	# infra-cache-numhosts: 10000
-+	infra-cache-numhosts: 200
- 	
- 	# define a number of tags here, use with local-zone, access-control.
- 	# repeat the define-tag statement to add additional tags.
-@@ -215,6 +228,8 @@ server:
- 	# access-control: ::0/0 refuse
- 	# access-control: ::1 allow
- 	# access-control: ::ffff:127.0.0.1 allow
-+	access-control: 0.0.0.0/0 allow
-+	access-control: ::0/0 allow
- 
- 	# tag access-control with list of tags (in "" with spaces between)
- 	# Clients using this access control element use localzones that
-@@ -309,12 +324,15 @@ server:
- 	#	positive value: fetch that many targets opportunistically.
- 	# Enclose the list of numbers between quotes ("").
- 	# target-fetch-policy: "3 2 1 0 0"
-+	target-fetch-policy: "2 1 0 0 0 0"
- 
- 	# Harden against very small EDNS buffer sizes.
- 	# harden-short-bufsize: no
-+	harden-short-bufsize: yes
- 
- 	# Harden against unseemly large queries.
- 	# harden-large-queries: no
-+	harden-large-queries: yes
- 
- 	# Harden against out of zone rrsets, to avoid spoofing attempts.
- 	# harden-glue: yes
-@@ -414,7 +432,7 @@ server:
- 	# you start unbound (i.e. in the system boot scripts).  And enable:
- 	# Please note usage of unbound-anchor root anchor is at your own risk
- 	# and under the terms of our LICENSE (see that file in the source).
--	# auto-trust-anchor-file: "@UNBOUND_ROOTKEY_FILE@"
-+	auto-trust-anchor-file: "@UNBOUND_ROOTKEY_FILE@"
- 
- 	# File with DLV trusted keys. Same format as trust-anchor-file.
- 	# There can be only one DLV configured, it is trusted from root down.
-@@ -504,15 +522,18 @@ server:
- 	# the amount of memory to use for the key cache.
- 	# plain value in bytes or you can append k, m or G. default is "4Mb".
- 	# key-cache-size: 4m
 +	key-cache-size: 100k
- 
- 	# the number of slabs to use for the key cache.
- 	# the number of slabs must be a power of 2.
- 	# more slabs reduce lock contention, but fragment memory usage.
- 	# key-cache-slabs: 4
-+	key-cache-slabs: 1
- 
- 	# the amount of memory to use for the negative cache (used for DLV).
- 	# plain value in bytes or you can append k, m or G. default is "1Mb".
- 	# neg-cache-size: 1m
 +	neg-cache-size: 10k
- 
- 	# By default, for a number of zones a small default 'nothing here'
- 	# reply is built-in.  Query traffic is thus blocked.  If you
++
++	# gentle on recursion
++	target-fetch-policy: "2 1 0 0 0 0"
++	harden-large-queries: yes
++	harden-short-bufsize: yes
++
++	# Enable a trust acnhor and modules "validator iterator." However, Unbound
++	# RFC5011 "auto-trust-anchor-" activity can be busy and harmful to flash ROM.
++	# "/etc/unbound" (directory & files) needs chown for write access. Else, use 
++	# plain "trust-anchor-" to treat the key file as static.
```

Comment:
Dnsmasq has a clever fix due to the exposure that program has to minimal hardware. There is a time stamp file (just touch) which is used to compare now-time, with file-time, and with dnsmasq-start-time. When unreconciled, DNSSEC glosses over the time requirement. Once reconciled (ntp update) the file is retouched, DNSSEC cache is flushed, and time is strictly part of new DNSSEC. Dnsmasq also has the ability to do the same with external SIGUSR. Unbound doesnt have this yet either way.

I will have more intuitive work arounds with UCI. But that needs to be a unique pull request. For this pull, the goal is a package that "works", does no harm, and with obvious perscriptive opions to enable more. All three of your listen options are expressed in the conf file deposited into /etc/unbound.